components/helpers/trace.js

```diff
@@ -0,0 +1,7 @@
+define('trace', [], () =>
```

Comment:
I think we can lose this.  we shouldn't be leaving these trace statements in the code once checked in.  we can all console.log while we develop, then remove them before checking in